lib/cli/watch.js

```diff
@@ -17,9 +18,16 @@ export default async function watch(options = {}) {
 
   const server = new Server(reptar);
   await server.start();
-
   activity.end(startActivity);
 
   process.stdout.write('\n');
   log.info('Server running at:', server.server.info.uri);
+
+  if (reptar.options.browserSync) {
```

Comment:
Did you test disabling browserSync? I think you'd want the options object given to the watch function in here.